pkg/sql/storageparam/storage_param.go

```diff
@@ -65,6 +65,9 @@ func Set(
 		// Cast these as strings.
 		expr := paramparse.UnresolvedNameToStrVal(sp.Value)
 
+		defer semaCtx.Properties.Restore(semaCtx.Properties)
```

Comment:
Could you add a comment explaining why they are not allowed?

Reply:
added